src/Ng/Jobs/MonitoringProcessorJob.cs

```diff
@@ -27,11 +26,16 @@ namespace Ng.Jobs
     /// </summary>
     public class MonitoringProcessorJob : LoopingNgJob
     {
+        private const int DefaultQueueLoopDurationHours = 24;
+        private const int DefaultEmptyQueueDelaySeconds = 30;
+
         private PackageValidator _packageValidator;
         private IStorageQueue<PackageValidatorContext> _queue;
         private IPackageMonitoringStatusService _statusService;
         private IMonitoringNotificationService _notificationService;
         private CollectorHttpClient _client;
+        private TimeSpan _queueLoopDuration;
```

Comment:
The default value is suffixed with the unit `Hours`. I'd propose to also add this suffix to this private field. --> `_queueLoopDurationHours`.
Same for `Seconds` and `_emptyQueueDelay` --> `_emptyQueueDelaySeconds'

Reply:
This is a `TimeSpan`, which means it could be used to get the duration in any value--seconds, hours, minutes, etc. For that reason, I kept the name of this field without any unit, and made the default value const and configuration parameter named with the unit (e.g. `DefaultQueueLoopDurationHours` and `-queueLoopDurationHours`) because they are actually parsed in a way specific to the unit.

Reply:
Oh, right! 👍